CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.14)
+
+include(cmake/prelude.cmake)
```

Comment:
```
list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
include(prelude.cmake)
```
 You can use include without relative paths with this small trick

Reply:
Awesome repository , helped me a lot